tests/python_tests/test_kv_cache_eviction.py

```diff
@@ -314,13 +311,22 @@ def test_kvcrush_vs_snapkv_baseline(device, test_struct):
 
     scheduler_config_kvcrush = get_scheduler_config(num_kv_blocks)
     scheduler_config_kvcrush.use_cache_eviction = True
-    scheduler_config_kvcrush.cache_eviction_config = KVCRUSH_TEST_CONFIG
+    max_cache_size, budget, anchor_mode = OPTIMAL_KVCRUSH_CONFIGS[subset]
+    config = CacheEvictionConfig(
+        start_size=32,
+        recent_size=128,
+        max_cache_size=max_cache_size,
+        aggregation_mode=AggregationMode.NORM_SUM,
+        apply_rotation=False,
+        snapkv_window_size=8,
+        kvcrush_config=KVCrushConfig(budget=budget, anchor_point_mode=anchor_mode)
+    )
```

Comment:
It looks like we can remove `KVCRUSH_TEST_CONFIG` now or replace `CacheEvictionConfig` to predefined `KVCRUSH_TEST_CONFIG`